src/test_typing_extensions.py

```diff
@@ -6617,6 +6617,22 @@ def test_allow_default_after_non_default_in_alias(self):
         a4 = Callable[[Unpack[Ts]], T]
         self.assertEqual(a4.__args__, (Unpack[Ts], T))
 
+    @skip_if_py313_beta_1
+    def test_generic_with_broken_eq(self):
+        # See https://github.com/python/typing_extensions/pull/422 for context
+        class BrokenEq(abc.ABCMeta):
```

Comment:
not specific to the `abc` module; any metaclass will do:

```suggestion
        class BrokenEq(type):
```